gridy/Sources/Presentations/PlanBoard/TimelineLayout/TimeAxisArea/TimeAxisAreaView.swift

```diff
@@ -8,31 +8,41 @@
 import SwiftUI
 
 struct TimeAxisAreaView: View {
-    // TODO: TimeAxisAreaView (날짜 및 공휴일 전역 선언, ObservableScrollView 해체, ZStack 월 Scroll 안되게)
-    let startDate = Date().formattedDate
+    // TODO: TimeAxisAreaView (날짜 및 공휴일 상위 뷰에 선언)
+    let startDate = Date()
     let numberOfDays = 200
     
     @State private var holidays: [Date] = []
```

Comment:
```suggestion
    @State private var holidays = [Date]()
```
이렇게 초기화하면 타입추론이 가능해지니 타입을 명시하지 않아도 되어요! 
지금 코드도 좋아서 요기 수정은 옵셔널!입니다!

Reply:
완료😇